src/components/abSpecific/UserListItem.tsx

```diff
@@ -29,17 +29,23 @@ export function UserListItem(props: Props) {
     onClick(userInfo)
   })
 
+  const username = userInfo.username ?? s.strings.missing_username
   return (
     <TouchableOpacity
+      accessible={false}
       style={styles.container}
       onPress={handlePress}
       onLayout={onLayout}
     >
-      <EdgeText style={styles.text}>
-        {userInfo.username ?? s.strings.missing_username}
+      <EdgeText accessible style={styles.text}>
+        {username}
       </EdgeText>
-      <TouchableOpacity onPress={handleDelete}>
+      <TouchableOpacity
+        testID={`${username}.deleteIcon`}
+        onPress={handleDelete}
+      >
         <MaterialIcon
+          accessible
```

Comment:
Since the icon is inside a `TouchableOpacity`, wouldn't the opacity be the Maestro target and the inner icon `accessible` be useless?